### PR TITLE
SDK update, Selectors update

### PR DESCRIPTION
**Pull request by OlehVeselov92**
- Updated selectors to relevant;
- Removed "merchantMetrics" and "reviewsLink" keys from the output object (this info is not represented on the page);
- Updated SDK to 1.2.0.

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -51,9 +51,13 @@ Apify.main(async () => {
...
+        launchContext: {
+            launchOptions: {
+                headless: true,
+                waitUntil: 'domcontentloaded',
+            },
+            useChrome: true,
+            stealth: true,
         },
         gotoFunction: async ({ request, page }) => {
             return page.goto(request.url, {
```
Comment (pocesar) [inline src/main.js]: `gotoFunction` is now `preNavigationHooks`. it should be 

```js
preNavigationHooks: [async ({}, gotoOptions) => { 
  gotoOptions.waitUntil = 'load';
  gotoOptions.timeout = 18000;
}]
```

File: src/main.js [hunk 2]
```diff
@@ -51,9 +51,13 @@ Apify.main(async () => {
         handlePageTimeoutSecs: 240,
         maxConcurrency: 20,
         proxyConfiguration,
-        launchPuppeteerOptions: {
-            timeout: 120 * 1000,
-            headless: true,
+        launchContext: {
+            launchOptions: {
+                headless: true,
+                waitUntil: 'domcontentloaded',
```
Comment (pocesar) [inline src/main.js]: `'domcontentloaded'` or `'load'`?

File: src/main.js [hunk 3]
```diff
@@ -51,9 +51,13 @@ Apify.main(async () => {
         handlePageTimeoutSecs: 240,
         maxConcurrency: 20,
         proxyConfiguration,
-        launchPuppeteerOptions: {
-            timeout: 120 * 1000,
-            headless: true,
+        launchContext: {
+            launchOptions: {
+                headless: true,
```
Comment (pocesar) [inline src/main.js]: `headless` or `useChrome`?

**Review by pocesar — COMMENTED** [review]
package-lock.json can be deleted (and .gitignored)
`productDetails` seems to be always `null`, not sure if it's a problem with the selector or it's like the 'merchantMetrics' issue (non-existent anymore) 
update the Dockerfile to use the latest chrome images `apify/actor-node-puppeteer-chrome`

overall, the resulting dataset looks good

**Comment by OlehVeselov92** [comment]
Made changes. Sorry for silly mistakes in crawler options) 
About  "productDetails":
It was originally null in the actor. 

In description on the page there is no full item description. Only first two sentences or so, and three dots.
Maybe that's why somebody decided to make it null.
Anyway, first two sentences is better then null) So i added it to the output.

File: .gitignore [hunk 4]
```diff
@@ -1,5 +0,0 @@
-apify_storage
```
Comment (pocesar) [inline .gitignore]: this file needs to stay 😄 
and add `package-lock.json` here

**Review by pocesar — CHANGES_REQUESTED** [review]
looks great now, just bring back `.gitignore` with the extra line 👍🏻

**Review by pocesar — APPROVED** [review]
🚀

**Pull request merged**